aws/common/outputs.tf

```
@@ -41,3 +41,11 @@ output "kms_arn" {
 output "lambda_ses_receiving_emails_arn" {
   value = aws_lambda_function.ses_receiving_emails.arn
 }
+
+output "aws_s3_bucket_asset_bucket_id" {
```

```suggestion
output "s3_bucket_asset_bucket_id" {
```
to follow the naming convention in this file